### ISSUE TITLE
Can we upgrade the Tensor Flow version to 2.X for this project? 

### ISSUE THREAD

**Issue by kandilidinesh** [xavysp/DexiNed#12]
As I executed the code, I was getting many deprecation warnings regarding the Tensor Flow 1.1X version. So, if possible, please let us know if this repository will be upgraded to Tensor Flow 2.X or can it be done with few modifications or it cannot be upgraded?

**Comment by xavysp** [comment]
Hi, 
just today I have installed TF2.0 maybe I will do it, but it will take a couple of months ( I mean to work with Keras layers), right now I am starting another project. There is another option by using tf.compat.v1. and disabling eager_execution tf.compat.v1.disable_eager_execution(), I think this last option is the easiest and fast way to run DexiNed  in TF2.0

**Comment by kandilidinesh** [comment]
Hi @xavysp sir, 

Yeah, I think I get it. I was gonna ask for migrating from Tensor Flow to Keras since we are not calculating Accuracy in this implementation but Keras has metrics for accuracy while fitting the model. Thank you for your reply.

**Comment by xavysp** [comment]
> Hi @xavysp sir,
> 
> Yeah, I think I get it. I was gonna ask for migrating from Tensor Flow to Keras since we are not calculating Accuracy in this implementation but Keras has metrics for accuracy while fitting the model. Thank you for your reply.

Hi really, I really didn't know abou it, could you share with me the link to know more about it?

**Comment by kandilidinesh** [comment]
https://www.tensorflow.org/api_docs/python/tf/keras/metrics/Accuracy

For your reference. 🙂
